app/src/main/java/com/cornellappdev/uplift/ui/screens/ProfileCreationScreen.kt

```diff
@@ -0,0 +1,227 @@
+        )
+    }) { innerPadding ->
+        val checkedState = remember { mutableStateOf(false) }
+        val checkedState2 = remember { mutableStateOf(false) }
+        val checkedState3 = remember { mutableStateOf(false) }
+
+        Column(
+            modifier = Modifier
+                .padding((innerPadding))
+                .fillMaxWidth(),
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Spacer(modifier = Modifier.height(46.dp))
+            //TODO - this Box placeholder will be replaced with a component to choose photo
+            Box(
+                modifier = Modifier
+                    .size(155.dp)
+                    .shadow(
+                        elevation = 20.dp,
+                        shape = CircleShape,
+                        ambientColor = GRAY01,
+                        spotColor = GRAY03
+                    )
+                    .background(color = GRAY01, shape = CircleShape)
+                    .border(width = 8.dp, color = Color.White, shape = CircleShape),
+                contentAlignment = Alignment.Center
+            ) {
+                Image(
+                    painter = painterResource(R.drawable.camera),
+                    contentDescription = "",
+                    modifier = Modifier.offset(y = 5.dp)
+                )
+            }
+            Text(
+                text = "First name Last name",
+                modifier = Modifier.padding(vertical = 24.dp),
+                fontSize = 24.sp,
+                fontFamily = montserratFamily,
+                fontWeight = FontWeight.Bold
+            )
+            Spacer(modifier = Modifier.height(25.dp))
+
+
+
+            Column(verticalArrangement = Arrangement.spacedBy(8.dp)) {
+                Row(verticalAlignment = Alignment.CenterVertically) {
+                    Checkbox(
+                        checked = checkedState.value,
+                        onCheckedChange = { checkedState.value = it },
+                        colors = checkboxColors
+                    )
+                    Text(
+                        text = "I agree with EULA terms and agreements",
+                        fontSize = 12.sp,
+                        fontFamily = montserratFamily,
+                        fontWeight = FontWeight.Normal
+                    )
+                }
+
+                Row(verticalAlignment = Alignment.CenterVertically) {
+                    Checkbox(
+                        checked = checkedState2.value,
+                        onCheckedChange = { checkedState2.value = it },
+                        colors = checkboxColors
+                    )
+                    Text(
+                        text = "I allow Uplift to access data on my gym usage",
+                        fontSize = 12.sp,
+                        fontFamily = montserratFamily,
+                        fontWeight = FontWeight.Normal
+                    )
+                }
```

Comment:
I see also that you repeatedly repeat this `Row` into `Checkbox` into `Text`. We should probably make a component--even if private and local to this file--for this.